### DISCUSSION TITLE
Sample python project repo for scientific projects

### DISCUSSION THREAD

**Discussion by machallboyd** [NREL/mappymatch#147] [Ideas]
A friend of a friend referred me to this new sample repository for scientific Python projects being distributed, geared at scientists who know zero practices but what to start up a project. (Industry programmers aren't always up to date on these concepts either, myself included.)

https://github.com/joaomcteixeira/python-project-skeleton

Seems like we've got some of the bases covered already, but I thought I would leave it here for people to look at.

**Comment by nreinicke** [comment]
Very cool, thanks for sharing. 

I do like the idea of:

> automated version bump with [bump2version](https://github.com/c4urself/bump2version), git tagging, and Python packaging to PyPI on Pull Request merge
